src/Murder.Editor/CustomFields/EnumField.cs

```diff
@@ -36,8 +36,9 @@ public override (bool modified, object? result) ProcessInput(EditorMember member
                 return (modified, Enum.ToObject(t, intValue));
             }
 
-            (modified, intValue) = ImGuiHelpers.DrawEnumField($"##{member.Name}", t, intValue);
-            return (modified, Enum.ToObject(t, intValue));
+            object enumValue;
+            (modified, enumValue) = ImGuiHelpers.DrawEnumField($"##{member.Name}", t, intValue);
```

Comment:
this should no longer be needed?

Reply:
This is the previous code, no longer needed